interview-questions.md

```diff
@@ -8,62 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: Parameters are used in creating and declaring a function. While arguments are used when invoking the function.
 
-  Researched answer:
+  Researched answer: Function parameters specify a single or list of variables when declaring a function. Function arguments are passing/assigning value to a declared function when invoked. 
```

Comment:
This is often confusing at first. Your description here of argument is perfect, however the parameter is a variable name for the value that is passed into the function.  Ex:
`const myFunction = (parameter) => {
   return "something"
}
myFunction(argument)